### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by weigo87**
changed spell installation to ADD_SPELL

### CODE REVIEW

File: SoS/Readme/SoS-Changelog.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
+#Version 2.1.3 (November 14, 2024)
+
+- changed spell installation to ADD_SPELL
```
Comment (Roberciiik) [inline SoS/Readme/SoS-Changelog.md:3]: Could you please add line indicating what exactly was changed in Alternative Soubar area ARE file?

File: SoS/Readme/SoS-Changelog.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
+#Version 2.1.3 (November 14, 2024)
```
Comment (Roberciiik) [inline SoS/Readme/SoS-Changelog.md:1]: Please add space between `#` and `Version`

File: SoS/Readme/SoS-Changelog.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
...
+
+- changed spell installation to ADD_SPELL
+
+#Version 2.1.2 (July 7, 2024)
```
Comment (Roberciiik) [inline SoS/Readme/SoS-Changelog.md:5]: Please add space between `#` and `Version`

File: SoS/SoS.tp2 [hunk 4]
```diff
@@ -820,16 +849,16 @@ INCLUDE ~%MOD_FOLDER%/Lib/install-itm.tpa~
...
-THEN BEGIN
-COPY ~%MOD_FOLDER%/Compat/%MOD_FOLDER%/SPWI429.spl~          ~override~
-  SAY NAME1 @20651
-  SAY UNIDENTIFIED_DESC @20652
-
-COPY ~%MOD_FOLDER%/Compat/%MOD_FOLDER%/SPRB005A.bam~          ~override~
-       ~%MOD_FOLDER%/Compat/%MOD_FOLDER%/SPRB005B.bam~          ~override~
-       ~%MOD_FOLDER%/Compat/%MOD_FOLDER%/SPRB005C.bam~          ~override~
-END
+//ACTION_IF NOT FILE_EXISTS ~data/TDD-RULE.BIF~ AND NOT FILE_EXISTS_IN_GAME ~tdd.mrk~ OR GAME_IS ~bg2ee eet~
```
Comment (Roberciiik) [inline SoS/SoS.tp2:852]: Why this is commented out?

File: SoS/SoS.tp2 [hunk 5]
```diff
@@ -1315,58 +1344,25 @@ BEGIN @21011
...
-			END
-			PATCH_IF xcoord = 830 BEGIN
-				SAY offset+0x04 @20080 // North Lyrar's Hold
-			END
-			PATCH_IF xcoord = 1721 BEGIN
-				SAY offset+0x04 @20076 // Narwhal
-			END
-		END
+        // Info points
+        SAY 0x257c @20075 // Social Club
```
Comment (Roberciiik) [inline SoS/SoS.tp2:1348]: Is it intended to restore a more "basic" way to assign info triggers?

Comment (weigo87) [reply SoS/SoS.tp2:1348]: I have to delete the pull request again, something went wrong :(

**Comment by weigo87** [comment]
@Roberciiik 
Somehow an old branch was still active and I overlooked it. I will sort all the files again and then it should work.

**Comment by Roberciiik** [comment]
Sure, I will review the next pull request then.

**Pull request closed without merging**